yarn.lock

```diff
@@ -16751,10 +16751,10 @@ path-key@^3.0.0, path-key@^3.1.0:
   resolved "https://registry.yarnpkg.com/path-key/-/path-key-3.1.1.tgz#581f6ade658cbba65a0d3380de7753295054f375"
   integrity sha512-ojmeN0qd+y0jszEtoY48r0Peq5dwMEkIlCOu6Q5f41lfkswXuKtYrhgoTpLnyIcHm24Uhqx+5Tqm2InSwLhE6Q==
 
-path-parse@^1.0.5, path-parse@^1.0.6:
-  version "1.0.6"
-  resolved "https://registry.yarnpkg.com/path-parse/-/path-parse-1.0.6.tgz#d62dbb5679405d72c4737ec58600e9ddcf06d24c"
-  integrity sha512-GSmOT2EbHrINBf9SR7CDELwlJ8AENk3Qn7OikK4nFYAu3Ote2+JYNVvkpAEQm3/TLNEJFD/xZJjzyxg3KBWOzw==
+path-parse@^1.0.6:
+  version "1.0.7"
+  resolved "https://registry.yarnpkg.com/path-parse/-/path-parse-1.0.7.tgz#fbc114b60ca42b30d9daf5858e4bd68bbedb6735"
+  integrity sha512-LDJzPVEEEPR+y48z93A0Ed0yXb8pAByGWo/k5YYdYgpY2/2EsOsksJrq7lOHxryrVOn1ejG6oAp8ahvOIQD8sw==
```

Comment:
This chunk is the only change that's necessary for this PR.

Reply:
pretty much. to bump path-parse, we needed the version of resolve 1.20.0 that also uses path-parse 1.0.7.

Reply:
All of the entries for `resolve` which depend on `path-parse` specify a version range of `^1.0.5` or `^1.0.6`, which means "any version above 1.0.6 and below 2.0.0" will match. `1.0.7` falls within that range, so the entries for `resolve` do not need to be changed.

* `^1.0.5` matches any version at or above `1.0.5` and below `2.0.0`
* `^1.0.6` matches any version at or above `1.0.6` and below `2.0.0`

A single entry for `path-parse@^1.0.5, path-parse@^1.0.6:` which resolves as version `1.0.7` is all that's needed.